tests/conftest.py

```diff
@@ -110,11 +109,10 @@ def finalizer():
         Delete the Ceph block pool
         """
         for instance in instances:
-            if not instance.is_deleted:
-                instance.delete()
-                instance.ocp.wait_for_delete(
-                    instance.name
-                )
+            instance.delete()
+            instance.ocp.wait_for_delete(
```

Comment:
Just thinking about if we can cover this in delete function as well to not have to do it in all finalizers?  Maybe we can achieve this with:
instance.delete(wait=true) and change mentioned in comment above above.
This case I think we can avoid to do this wait_for_delete here in finalizers. WDYT?

Reply:
I thought about this one as I was refactoring the code, the wait seems to be default in *most*cases, but it needs all tier1 testing and some more digging into helpers code, I will file an issue and we can take this up in different PR.

Reply:
I agree with @vasukulkarni let's handle that change independently from `is_deleted`.